assets/root-files/Authentication/README.md

```diff
@@ -0,0 +1,8 @@
+# zend-authentication
+
+The `Zend\Authentication` component provides an API for authentication and
```

Comment:
After the splits are done, sure. Not as part of the move, however (too many moving targets).